AmazonChimeSDKDemo/Podfile

```diff
@@ -4,12 +4,12 @@ platform :ios, '11.0'
 target 'AmazonChimeSDKDemoPods' do
   use_frameworks!
   # Pods for AmazonChimeSDKDemoPods
-  pod 'AmazonChimeSDK-Bitcode', 'AMAZON_CHIME_SDK_VERSION'
```

Comment:
Won't this break Jenkins? Iirc, those are the placeholders for the target SDK version to build.

Reply:
Sigh. OK I'll get this part out of this PR.